README.md

```diff
@@ -47,26 +56,26 @@ $ npx prisma migrate diff \
   --output migrations/0001_create_tables.sql
 ```
 
-4. Generate the necessary Prisma files for Zod validations and any other configured Prisma outputs:
+5. Generate the necessary Prisma files for Zod validations and any other configured Prisma outputs:
 
 ```sh
 $ npx prisma generate
 ```
 
-5. Apply the migration to your local database:
+6. Apply the migration to your local database:
 
 ```sh
-$ npx wrangler@latest d1 migrations apply cdph-starter
```

Comment:
Removing `latest` since `npx` will use the version listed in the `package.json` anyways.